app/points/domain/workload.js

```diff
@@ -20,6 +20,8 @@ class Workload {
     this.licenseTiers = licenseTiers
     this.licenseCasesLast16Weeks = licenseCasesLast16Weeks
     this.communityCasesLast16Weeks = communityCasesLast16Weeks
+    this.stagingId = stagingId
+    this.workloadReportId = workloadReportId
```

Comment:
There should be a unit test that makes sure when this is set through the constructor it is assigned to the correct internal variable.